draft-ietf-acme-acme.md

```diff
@@ -1943,22 +1950,22 @@ url (required, string):
 : The URL to which a response can be posted.
 
 status (required, string):
-: The status of this authorization.  Possible values are: "pending", "valid",
+: The status of this challenge.  Possible values are: "pending", "valid",
 and "invalid".
 
 validated (optional, string):
-: The time at which this challenge was completed by the server, encoded in the
+: The time at which the server considered this challenge complete, encoded in the
```

Comment:
Is "complete" the best word for a field called "validated"? I would think it would be "The time at which the server validated this challenge" or "The time at which the server validated this completed challenge".